clang/test/CodeGen/ptrauth-cpu-feature.c

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: aarch64-registered-target
+
+// Test that features requiring FEAT_PAuth fail early if the requirement is not met:
+//
+// RUN: not %clang %s -S -o - -target aarch64-linux-gnu -mcpu=cortex-a72 -mbranch-protection=pauthabi 2>&1 | FileCheck %s --check-prefix=FAIL
```

Comment:
It looks like a more generic target triple should be enough (aarch64-elf or even just aarch64).

Reply:
Some time ago I had a revision where MaskRay asked me to use a generic triple: see the comment https://reviews.llvm.org/D156882?id=546424#inline-1518496. I agree that specific triples are better for reproduceability, but is it OK in our particular context that running commands with different aarch64 triples gives different results? If results in all the cases must be equivalent, having a generic triple looks clearer IMHO.